src/test/test.js

```diff
@@ -45,7 +45,7 @@ describe('Registrar', () => {
       // fs.writeFileSync('src/test/contracts.json', JSON.stringify(compiled));
       // Use to speed up the testing process during development:
       const compiled = JSON.parse(fs.readFileSync('src/test/contracts.json').toString());
-      const deployer = compiled['DeployENS']; // eslint-disable-line
+      const deployer = compiled[':DeployENS']; // eslint-disable-line
```

Comment:
@Arachnid I think I've asked you this before, but can't find it. What does the `:` prefix do?